pkg/download/catalog.go

```diff
@@ -21,29 +21,34 @@ type catalogRes struct {
 }
 
 // CatalogHandler implements GET baseURL/catalog
-func CatalogHandler(dp Protocol, lggr log.Entry, eng *render.Engine) buffalo.Handler {
+func CatalogHandler(dp Protocol, lggr log.Entry) http.Handler {
```

Comment:
Content-type json should be in the header.

Reply:
@manugupt1 done